pandas/tests/groupby/test_sample.py

```diff
@@ -142,3 +142,10 @@ def test_groupby_sample_with_selections():
     result = df.groupby("a")[["b", "c"]].sample(n=None, frac=None)
     expected = DataFrame({"b": [1, 2], "c": [1, 2]}, index=result.index)
     tm.assert_frame_equal(result, expected)
+
+
+def test_groupby_sample_with_empty_inputs():
+    result = DataFrame({"a": [], "b": []})
+    expected = result.groupby("a").sample()
```

Comment:
nit: `result` should be the result that is being tested, expected should be what we expect. Can you switch these?